geo/src/main/java/org/odk/collect/geo/selection/SelectionMapFragment.kt

```diff
@@ -350,13 +350,7 @@ class SelectionMapFragment(
 
         for (item in items) {
             val point = MapPoint(item.latitude, item.longitude)
-            val featureId = map.addMarker(point, false, MapFragment.BOTTOM)
-
-            map.setMarkerIcon(
-                featureId,
-                if (featureId == selectedFeatureViewModel.getSelectedFeatureId()) item.largeIcon else item.smallIcon
-            )
-
+            val featureId = map.addMarker(point, false, MapFragment.BOTTOM, item.smallIcon)
```

Comment:
This ends up working because currently `onFeatureClicked` always ends up getting called after this. See my note at https://github.com/getodk/collect/pull/5099/files#r849054759